### ISSUE TITLE
Looking to front-end video files

### ISSUE THREAD

**Issue by bigdwg71** [alfredh/baresip#761]
I read through the documentation but couldn't seem to find what I was looking/hoping for... Does baresip have a mode of operation that would allow me to front-end video files? Basically, call a SIP URI that lands on baresip and barsip negotiates one-way video from barsip to endpoint that originated the call?

Looking to be able to play out mostly mp4, maybe an mkv file. I see there is an ffmpeg-baresip library but i couldn't find any config examples as to how to leverage it to play local media files.

Basically, can we setup baresip to "host" a set of URIs in baresip that result in a local media file being played out to the remote calling party.

Any plugin/packages that would enable that?

**Comment by alfredh** [comment]
yes, you can use the avformat module as a video source

for example:

```
video_source    avformat,test.mp4
```

**Comment by alfredh** [comment]
the Issues tracker is mainly for reporting defects.

Please post to the re-devel mailing list if you have additional questions:

http://lists.creytiv.com/mailman/listinfo/re-devel
